Src/DependencyCollector/Shared/Implementation/DependencyCollectorEventSource.cs

```diff
@@ -267,8 +267,12 @@ private string GetApplicationName()
             try
             {
 #if NETCORE
+#if NETSTANDARD1_3
+                name = "Undefined"; // netstandard1.3 does not have API to retrieve Assembly/AppDomain name
```

Comment:
Getting assembly name with reflection now.
Even though API is not available on netstandard1.3, the method will be there anyway in runtime with .NET 4.0+ or netcoreapp1.0+